### PR TITLE
OCPBUGS-14623: Replaced word "wrong" with "non-recovery" for non-recovery hosts

### PR DESCRIPTION
**Pull request by palonsoro**
OCPBUGS-14623: Replaced word "wrong" with "non-recovery" for non-recovery hosts

Version(s):
4.11 to latest

Issue:
https://issues.redhat.com/browse/OCPBUGS-14623 

Link to docs preview:

https://61115--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

QE review:
- [x] QE has approved this change.

Additional information:

(none)

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
Looks good to me!

**Comment by anuragthehatter** [comment]
/lgtm

Thanks

**Comment by palonsoro** [comment]
@adellape the PR you mention has introduced a **severe** error here:

https://github.com/openshift/openshift-docs/blob/e26fd82e9ee9d47581fed251879b7617434c5460/modules/dr-restoring-cluster-state.adoc?plain=1#L308

Nodes cannot be optionally deleted and re-created. Nodes **must** stay deleted or the OVN-Kubernetes control plane won't be redeployed. The resolution of https://issues.redhat.com/browse/OCPBUGS-6728 may lead to extra details.

I am opening **urgent** bug report for this and fix it myself. As this bug is less urgent, I'll wait for the other one before rebasing and asking to merge this one.

**Comment by palonsoro** [comment]
Once https://github.com/openshift/openshift-docs/pull/61352 fixed the problem I described above, I could rebase this on top of that fix.

/label merge-review-needed

**Comment by kcarmichael08** [comment]
@palonsoro - Everything went OK except for the 4.10 cherry pick. That will have to be manually cherry picked to resolve merge conflicts - [instructions](https://docs.google.com/document/u/2/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub#h.dsrvo6g6qbtl) here.

**Comment by palonsoro** [comment]
Thx @kcarmichael08 . 

I am sorry but it seems I made a mistake with the versions. This PR fixes some stuff that didn't make it to 4.10 docs.

So I'll remove 4.10 from the bug and fix the PR description. Sorry.

**Pull request merged**
